Project-I-Social-Card/social-card/src/components/CardComponents/Card.css

```diff
@@ -1 +1,16 @@
 /* All styles for your Card Components live here */
```

Comment:
I would also take a look at your index.css and add a constraint to your width with max-width and then assign width 100% so it helps to control the width in the smaller divs. Try to make your styling look a little more like the design file. Right now text-size is a little too big and the proportions are off on your images. Being pixel perfect will go a long way in helping you to become a UI developer.